bacon.toml

```diff
@@ -122,7 +125,8 @@ b = "job:build"
 
 [jobs.build-swc]
 command = [
-	"cargo", "build", "-p", "swc_isograph_plugin", "--release", "--target", "wasm32-wasip1", 
+	"cargo", "build", "-p", "swc_isograph_plugin", "--release", "--target", "wasm32-wasip1",
 	"--color", "always",
 ]
+watch = ["crates"]
```

Comment:
good catch... this should probably be `-p crates` instead of just watching the swc_isograph_plugin folder